Demo/Demo/SwiftUIComponents/CardPaymentViews/CardApprovalResultView.swift

```diff
@@ -0,0 +1,38 @@
+import SwiftUI
+
+struct CardApprovalResultView: View {
+
+    @ObservedObject var cardPaymentViewModel: CardPaymentViewModel
+
+    var body: some View {
+        switch cardPaymentViewModel.state.approveResultResponse {
+        case .idle, .loading:
+            EmptyView()
+        case .loaded(let approveResultResponse):
+            getSuccessView(approveResultResponse: approveResultResponse)
+        case .error(let errorMessage):
+            ErrorView(errorMessage: errorMessage)
+        }
+    }
+
+    func getSuccessView(approveResultResponse: CardPaymentState.CardResult) -> some View {
```

Comment:
```suggestion
    func getSuccessView(approvalResult: CardPaymentState.CardResult) -> some View {
```

Reply:
I think you are looking at older version. I already changed this.

Reply:
Ope - you are right :)